microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -36,7 +36,7 @@ See also [Optimize website speed using Lighthouse](../speed/get-started.md).
 
 In the following tutorial, you open DevTools on a "Sluggish Animation" demo page and use the **Performance** tool to find a performance bottleneck on the page.
 
-1. Open Microsoft Edge in **InPrivate Mode**.  InPrivate Mode ensures that Microsoft Edge runs in a clean state.  For example, if you have a lot of extensions installed, the extensions may create noise in your performance measurements.
+1. Open Microsoft Edge in **InPrivate Mode**.  InPrivate Mode ensures that Microsoft Edge runs in a clean state.  For example, if you have numerous extensions installed, the extensions may create noise in your performance measurements.
 
    <!--TODO: replace section when updated for Chromium-based Edge  -->
 
```

Comment:
Improve the presentation of the link to the demo page and make it more useful.  Provide link text.  (Note that the tab text != the page title):
```
1. Open the [devtools-performance-get-started](https://microsoftedge.github.io/Demos/devtools-performance-get-started/) demo page in a new tab or window.  The **Sluggish Animation** tab opens.

   The source code is at [MicrosoftEdge / Demos > devtools-performance-get-started](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-performance-get-started).
```